### ISSUE TITLE
Mean High Water (MHW) tasks

### ISSUE THREAD

**Issue by clintonlunn** [FREAC/LABINS#216]
- MHW is text only right now.  Implement a feature service for the Mean High Water data into the interactive LABINS map. 
- Implement the Mean High Water features into the Data Query panel.

**Comment by shodge17** [comment]
need a new section in the information Panel for the MHW surveys.  There is no layer for these features.  The strategy will be to use the Township-Range-Section layer as the block of code to build the needed information.  Once in the block build a new section of the info panel and call it MHW Surveys.  All we are going to do is build a link similar to what you see in the GLO section.  the link needs to look like this:
<a target="_blank" href=https://www.labins.org/survey_data/water/water.cfm?town1=' + results[i - 1].attributes.tr_dissolve.substring(0, 2) + '&town2=' + results[i - 1].attributes.tr_dissolve.substring(2, 3) + '&range1=' + results[i - 1].attributes.tr_dissolve.substring(3, 5) + '&range2=' + results[i - 1].attributes.tr_dissolve.substring(5, 6) + '&sec=' + results[i - 1].attributes.tr_dissolve.substring(1,2) + '>' + 'Search for MHW Surveys in this section ' + '</a><br>' 

Let me know if you have questions about this one.

**Comment by shodge17** [comment]
not sure why, but the section part of the link I gave you is not right.  Here is what should work:
attributes.twnrngsec.substring(8, )

**Comment by shodge17** [comment]
I don't see this update in the master right now.  Agreed?  Feature zoom to 38S R41E section 5.  Just click in the section.  You will not see the MHW section in the info panel.  I think your intent was to have all of this issue in the pull I just did right?

**Comment by clintonlunn** [comment]
> I don't see this update in the master right now.  Agreed?  Feature zoom to 38S R41E section 5.  Just click in the section.  You will not see the MHW section in the info panel.  I think your intent was to have all of this issue in the pull I just did right?

Yes that was my intent, I must have missed this change. Thanks for pointing it out. I'll get a pr up for this.
